### ISSUE TITLE
Firefox editableCellTemplate Issue

### ISSUE THREAD

**Issue by zlaja1983** [angular-ui/ui-grid#1195]
In Firefox, if the editableCellTemplate is of:

\<input type="number" ng-class="\'colt\' + col.index" ng-input="COL_FIELD" ng-model="COL_FIELD" />

The first time a user clicks, it allows him to modify an entry. Any subsequent attempts, it does not enter the edit mode. We are currently using: version 2.0.7 of the grid. It is worth noting that it works in Chrome and IE without an issue.

There is an error in the Firefox console once trying to edit the same cell:
TypeError: current is null angular.js:9186

Here is a jsfiddle to illustrate the problem in Firefox: Try modifying the 'age' column.

http://jsfiddle.net/zlaja1983/38hPb/

**Comment by EvanWinstanley** [comment]
I know this is an old issue so you guys might have moved on, but I was able to resolve this issue with a simple `$timeout` (see pull request above).

Basically for reasons I couldn't quite figure out, Firefox was firing a blur and then a focus when editing a cell with `<input type=number />`. Adding the timeout resolved the issue. I tested for `enableCellEditOnFocus=true|false` (single click and double click editing) in FF and Chrome and everything seemed to still be working correctly in both browsers.

Hope this helps someone.
